src/main/kotlin/skytils/skytilsmod/features/impl/misc/ContainerSellValue.kt

```diff
@@ -109,18 +110,45 @@ object ContainerSellValue {
         }
     }
 
-    private fun shouldRenderOverlay(chestName: String): Boolean {
+    private fun isChestNameValid(chestName: String): Boolean {
         return (!inDungeons && (chestName == "Chest" || chestName == "Large Chest"))
                 || chestName.contains("Backpack")
                 || chestName.startsWith("Ender Chest (")
                 || (chestName.contains("Minion") && SBInfo.mode == SkyblockIsland.PrivateIsland.mode)
```

Comment:
idk how I didn't catch this before but I think this would trigger in the crafting recipe for minions